distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestConfiguration.java

```diff
@@ -710,14 +732,23 @@ public void testExportMetacards() throws Exception {
 
         console.runCommand(CATALOG_REMOVE_ALL_COMMAND, new RolePrincipal("admin"));
 
-        console.runCommand(String.format("%s %s",
+        console.runCommand(String.format("%s \"%s\"",
                 CATALOG_INGEST_COMMAND,
                 getDefaultExportDirectory().resolve("ddf.metacards")),
                 new RolePrincipal("admin"));
 
         assertMetacardsIngested(metacardIds.size());
     }
 
+    /**
+     * The felix file installer keeps open file handles on files and directories in the etc directory on Windows.
+     * This prevents files and directories from being deleted in some of the TestConfiguration itests when running 
+     * the itests on Windows. This method stops the felix file installer which releases the file handles.
+     */
+    private void closeFileHandlesInEtc() throws BundleException {
+        getServiceManager().stopBundle(FELIX_FILE_INSTALLER);
```

Comment:
I only found one call of `resetInitialState()` without `closeFileHandlesInEtc()` so it could have been considered for putting in one method. Did you consider merging into one a single method?